setup.cfg

```diff
@@ -13,6 +13,7 @@ classifiers =
     Programming Language :: Python :: 3.8
     Programming Language :: Python :: 3.9
     Programming Language :: Python :: 3.10
+    Programming Language :: Python :: 3.11
```

Comment:
It would be good to remove 3.7 (EOL) and add 3.12.

Reply:
Done.